src/features/workflow/modal/DeployModal.tsx

```diff
@@ -1,35 +1,38 @@
-import React, { useState, useEffect } from 'react'
-import { useLocation, useHistory } from 'react-router-dom'
+import React, { useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
 import { useDispatch, useSelector } from 'react-redux'
 import ReactCanvasConfetti from 'react-canvas-confetti'
 import classNames from 'classnames'
 
-import Button, { ButtonType } from '../../../chrome/Button'
-import Icon from '../../../chrome/Icon'
+import Button from '../../../chrome/Button'
 import { clearModal, setModalLocked } from '../../app/state/appActions'
-import IconButton from '../../../chrome/IconButton'
-import TextInput from '../../../chrome/forms/TextInput'
-import Checkbox from '../../../chrome/forms/Checkbox'
+
 import { deployWorkflow } from '../../deploy/state/deployActions'
-import { workflowResetDryRunId, setWorkflowRef, workflowResetEditedClearedCells } from '../state/workflowActions'
+import { workflowResetDryRunId, workflowResetEditedClearedCells } from '../state/workflowActions'
 import {
   selectWorkflow,
   selectWorkflowQriRef,
   selectWorkflowDataset,
   selectLatestDryRunId,
   selectLatestRunId
 } from '../state/workflowState'
-import { validateDatasetName } from '../../session/state/formValidation'
 import RunStatusIcon from '../../run/RunStatusIcon'
 import { selectDeployRunId, selectDeployStatus } from '../../deploy/state/deployState'
-import { selectSessionUser } from '../../session/state/sessionState'
-import WarningDialog from '../WarningDialog'
 import { removeEvent } from "../../events/state/eventsActions"
-import { useDebounce } from "use-debounce"
 
-const DEBOUNCE_TIMER = 500
+export interface DeployModalProps {
+  username: string
+  name: string
+  runNow: boolean
+  isNew: boolean
+}
 
-const DeployModal: React.FC = () => {
+const DeployModal: React.FC<DeployModalProps> = ({
```

Comment:
I could go for a name change here, since this is being used after commits and deploy's. `CommitModal`? `ConfirmCommitModal`?

Reply:
Renamed to AutomationCommitModal